cryptostore/data/influx.py

```diff
@@ -22,9 +22,11 @@ def __init__(self, config: dict):
         self.data = None
         self.host = config.host
         self.db = config.db
-        self.addr = f"{config.host}/write?db={config.db}"
+        self.username = config.username if "username" in config.keys() else None
```

Comment:
you dont need to check config.keys() you can just do

`self.username = config.username if "username" in config else None`

or 

`self.username = config.get('username', None)`

You'd have to confirm the 2nd option, but I believe it would work